tasks/sanity-inspect-image.yaml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: sanity-inspect-image
+spec:
+  description: >-
+    Get manifest data for the source image and its base image to workspace
+  params:
+    - name: SOURCE_IMAGE
+      description: the fully qualified image name
+      default: quay.io/hongweiliu/conftest_oci:latest
```

Comment:
Shouldn't we remove a default? Especially if it's your personal repo

Reply:
Thanks. I thought someone would test it with one example. Let me remove it.